### DISCUSSION TITLE
Could I use SwiftGodot to add VoiceOver (iOS screen reader) support to a Godot project?

### DISCUSSION THREAD

**Discussion by oliver-dew** [migueldeicaza/SwiftGodot#110] [Q&A]
I have a simple Godot game where the user can tap on objects. The objects and the camera are both static. I would like to enable VoiceOver support and was wondering whether SwiftGodot or SwiftGodotKit could help with this.
I would need to be able to define accessibility elements on the `UIView` that hosts the Godot scene, is that exposed anywhere in SwiftGodot/ SwiftGodotKit? 

I would also need to find out the screen coordinates, accessibility labels, and a hook to the action of the tappable nodes. I guess I would create an `AccessibilityNode` custom class that would contain the label and some kind of hook to the action, and project the corners of its sibling's AABB back to the camera to get the 2D screen bounds of the element.

Does this sound like it could be possible?

**Comment by migueldeicaza** [comment]
They can, but it is not the most straightforward path.  Sorting out the core details of how to make this work doesn’t need swift Godot

the other challenge is your approach, which is whether swiftgodot/kit can be embedded in a UIview, and that’s not currently the case, so it this needs to be sorted out first
